multichain-aggregator/multichain-aggregator-server/tests/chains.rs

```diff
@@ -0,0 +1,128 @@
+mod helpers;
+
+use blockscout_service_launcher::test_server;
+use multichain_aggregator_logic::{repository::chains, types::chains::Chain};
+use multichain_aggregator_proto::blockscout::multichain_aggregator::v1 as proto;
+use pretty_assertions::assert_eq;
+use reqwest::StatusCode;
+use serde_json::json;
+use wiremock::{
+    matchers::{method, path},
+    Mock, MockServer, ResponseTemplate,
+};
+
+#[tokio::test]
+#[ignore = "Needs database to run"]
+async fn test_fetch_chains() {
+    let db = helpers::init_db("test", "startup_works").await;
```

Comment:
i think it's copypaste error

btw you can take a look athttps://docs.rs/function_name/latest/function_name/

Reply:
Used a `database!` macro from service launcher